core/templates/dev/head/editor/EditorServices.js

```diff
@@ -340,13 +340,12 @@ oppia.factory('changeListService', [
      * is correctly formed
      *
      * @param {object} gadgetData The dict containing new gadget information.
-     * @param {string} panelName The panel where the gadget is being added.
      */
-    addGadget: function(gadgetData, panelName) {
+    addGadget: function(gadgetData) {
       this._addChange({
         cmd: CMD_ADD_GADGET,
         gadget_dict: gadgetData,
-        panel_name: panelName
+        panel_name: gadgetData['panel']
```

Comment:
Confirming refactored all usage of panel_name to gadget_panel consistently across the backend.